bank-sdk/example-app/build.gradle.kts

```diff
@@ -23,15 +24,15 @@ plugins {
 
 android {
     // after upgrading to AGP 8, we need this (copied from the module's AndroidManifest.xml
-    namespace = "net.gini.android.capture.screen"
+    namespace = "net.gini.android.bank.screen"
```

Comment:
[Request] I looked at the Health SDK's example app and it's package name is `net.gini.android.health.sdk.exampleapp`. Let's follow that pattern and use `net.gini.android.bank.sdk.exampleapp` package name for the Bank SDK's example app.

Reply:
Done.